### DISCUSSION TITLE
Power switch feature

### DISCUSSION THREAD

**Discussion by NicoSjokk** [IT2901Nordic2024/project#5] [General]
Should the user actively use the power switch as a feature?
We've been discussing this internally. 

For instance:
    
As a user I've been tracking my time studying at my desk and am ready to go home. Would the intended functionality be for me to put the device in my backpack whilst it's turned on or that I as a user would turn it off with the power switch and only power it when I want to track a specific activity?

The reason we're asking this is because it affects how the device would register GPS data for instance in the future as well as some of the handling of the gyroscope/accelerometer.

**Comment by coderbyheart** [comment]
Yes, a powered-off mode makes sense to have. The easiest way is literally to turn it off.
